data/scripting/training_wheels/init.lua

```diff
@@ -16,7 +16,7 @@
 -- Their names need to be identical with their base file name (without the ``.lua`` file extension).
 -- For blocking the execution of a Training Wheel until the player has learned a prerequisite
 -- Training Wheel, list them in the Training Wheel's Lua table in the format
--- ``example = { "prerequisite1", "prerequisite1" },``.
+-- ``example = { descname = "Example", dependencies = "prerequisite1", "prerequisite1" } },``.
```

Comment:
No i18n markup for `descname` in the example?

Reply:
No, because the translation system would pick it up and put the string on Transifex.